src/Data/Fin/Induction.agda

```diff
@@ -94,24 +96,6 @@ private
   ... | no  n≢i = subst P (inject₁-lower₁ i n≢i) (Pᵢ₊₁⇒Pᵢ _ Pᵢ₊₁)
     where Pᵢ₊₁ = induct (rec _ (ℕ.≤-reflexive (cong suc (sym (toℕ-lower₁ i n≢i)))))
 
-------------------------------------------------------------------------
--- Induction over _≺_
-
-≺-Rec : RecStruct ℕ ℓ ℓ
```

Comment:
Deprecation missing from CHANGELOG?

Reply:
I've pushed a `CHANGELOG` entry, but it is not an orthodox '`x` deprecated in favour of `y` message'. Please advise.
Also, starting to panic lest the merge problems on other branches might have polluted this one, too. Ditto re advice!

Reply:
Nope, all looks good to me :+1: